weather/stations/davis.py

```diff
@@ -290,29 +353,31 @@ def _post_unpack(self, items):
     (('Index', 'B'), ('Records', '260s'), ('unused', '4B'), ('CRC', 'H')),
     order='=')
 
+
+class _TimeStruct(Struct):
+    FMT = (
+        ('Sec', 'B'),
+        ('Min', 'B'),
+        ('Hour', 'B'),
+        ('Day', 'B'),
+        ('Month', 'B'),
+        ('Year', 'B'),
+        ('CRC', 'H'),
+    )
+
+    def __init__(self):
+        super(_TimeStruct, self).__init__(self.FMT, '=')
+
+    def _post_unpack(self, items):
+        items['Year'] = items['Year'] + 1900
+        return items
+
+
 # init structure classes
 LoopStruct = LoopStruct()
 ArchiveAStruct = _ArchiveAStruct()
 ArchiveBStruct = _ArchiveBStruct()
-
-
-def _fields_to_weather_point(fields: dict) -> WeatherPoint:
-    """Convert VantagePro fields dictionary to WeatherPoint.
-
-    Only supports limited subset of data available in self.fields -
-    generally only those useful for posting to weather services.
-    """
-    return WeatherPoint(
-        temperature_f=fields['TempOut'],
-        pressure=fields['Pressure'],
-        dew_point_f=fields['DewPoint'],
-        humidity=fields['HumOut'],
-        rain_rate_in=fields['RainRate'],
-        rain_day_in=fields['RainDay'],
-        time=dt.datetime.strptime(fields['DateStampUtc'], "%Y-%m-%d %H:%M:%S"),
-        wind_speed_mph=fields['WindSpeed10Min'],
-        wind_direction=fields['WindDir'],
-    )
+timeStruct = _TimeStruct()
```

Comment:
this instance is not being used at the moment since in #5 is used for the two methods needed for getting and setting the console time. Will add those in a future contribution.